ngx-fudis/projects/ngx-fudis/src/lib/components/link/link.component.ts

```diff
@@ -67,12 +67,12 @@ export class LinkComponent implements AfterViewInit {
    * Option to create an external link to point a target page on another domain.
    * External link contains external icon and assistive aria-label
    */
-  @Input() isExternalLink: boolean = false;
+  @Input() external: boolean = false;
 
   /**
    * Link color
    */
-  @Input() color: 'primary-dark' | 'default' | 'white' = 'primary-dark';
+  @Input() color: 'default' | 'gray' | 'white' = 'default';
```

Comment:
These color names originate from `icon.component.scss` and this "breaks" external link (at least). I agree these are not intuitive names but let's do the name refactoring in a separate PR. I added a TODO comment for the class names and reverted your changes regarding on these.